examples/logistic_reg/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "logistic_reg"
+version = "0.1.0"
+authors = ["MesaTEE Authors <zhangyiqun01>"]
```

Comment:
We use unified authors: `authors = ["MesaTEE Authors <developers@mesatee.org>"]`.

Later, we will list all contributors.